app/package.json

```diff
@@ -36,6 +36,8 @@
     "json-source-map": "^0.6.1",
     "lodash": "^4.17.21",
     "match-sorter": "^6.3.1",
+    "micromark": "^4.0.0",
+    "micromark-extension-gfm": "^3.0.0",
```

Comment:
Does that use GitHub-falvored markdown?

Reply:
It is supposed to, in case the gfm extension is passed to micromark initialization, which it is in the code.
Have you found a use case where it doesn't?

Reply:
No. I was just wondering whether codeblocks could have e.g., a grey box to separate them from the rest of the text.

Reply:
@MarcoKorinth The comment
> I was just wondering whether codeblocks could have e.g., a grey box to separate them from the rest of the text.

is IMHO still open.

Reply:
Yes you're right.
Looks like the gfm plugin is missing this feature.
I think this should be possible by using a custom html component.
I will look into it.

Reply:
As discussed - this will be handled as separate issue #522